Sources/BraintreeVenmo/BTConfiguration+Venmo.swift

```diff
@@ -26,4 +26,9 @@ extension BTConfiguration {
     var venmoEnvironment: String? {
         json?["payWithVenmo"]["environment"].asString()
     }
+
+    /// Indicates whether ECD is enabled for the Venmo merchant.
```

Comment:
What does "enriched customer data" mean/do?

Reply:
This is a checkbox that lives in the Control Panel. When checked, it allows merchants to receive user info (name, email, and phone number as of now) during the consent flow.
In the context of this PR, merchants are being allowed to pass 2 flags - `collectCustomerBillingAddress` and `collectCustomerShippingAddress`. These addresses can only be collected when this ECD checkbox is checked in the CP.